.devcontainer/Dockerfile

```diff
@@ -0,0 +1,5 @@
+ARG VARIANT
```

Comment:
Any way I can test this?  I've never used .devcontainer - perhaps github codespaces is the easiest way?  Looks good from a birds-eye view.

Reply:
You can test this "locally" with Vscode (CPU/GPU), locally with Codespace, or in the browser with Github with Codespace top right button in this page (CPU only instance)

Reply:
I don't have VSCode - and I'm not in the codespaces beta.  For now this looks fine - if it works for you it looks good to me.

Reply:
Ok but Codespace is not in beta.
Probably you have an Org account and you Org has not activated Codespaces.

With your personal Github account you will find the button (top right)